docs/source/conf.py

```diff
@@ -113,6 +113,7 @@
         f'--project-version={version}',
         '--docformat=epytext',
         '--project-url=../customize.html',
+        '--theme=base',
```

Comment:
It would be useful to consider the `base` theme as theme in itself such that the custom themes can drop the bootstrap dependency without having to override the `bootstrap.min.css` file.

Reply:
I agree with having a base theme directory internally, but I was wondering if it should be filtered out of the themes list (for example by renaming it `_base`) as presented to the user.

Reply:
Using the `base` theme directly would be useful if we don't want bootstrap CSS and that we just want to add more custom CSS